frontend/server/src/DAO/Identities.php

```diff
@@ -534,14 +534,16 @@ public static function countActiveUsersByGender(
         int $startTimestamp,
         int $endTimestamp
     ): array {
+        $start = gmdate('Y-m-d H:i:s', $startTimestamp);
+        $end   = gmdate('Y-m-d H:i:s', $endTimestamp);
```

Comment:
Ya tenemos una función que convierte a timestamp de mysql:
```suggestion
        $start = \OmegaUp\DAO\DAO::toMySQLTimestamp($startTimestamp);
        $end   = \OmegaUp\DAO\DAO::toMySQLTimestamp($endTimestamp);
```